int32 number_of_obstacles
float64[] center_x
float64[] center_y
float64[] velocity_x
float64[] velocity_y
float64[] speed
float64[] radius